extensions/spring-cloud.md

```diff
@@ -1,51 +1,89 @@
+* `ConsistentHashChangeListener` - Adding a consistent hash change listener provides you the opportunity to perform a specific task if new nodes have been added to the known command handlers set.
 
-Other optional parameters for the `SpringCloudCommandRouter` are:
+> **Differing Command Capabilities per Node**
+>
+> It is not required that all nodes have the same set of command handlers. 
+> You may use different segments for different command types altogether. 
+> The Distributed Command Bus will always choose a node to dispatch a command to that has support for that specific type of command.
 
-* A "service instance filter" of type `Predicate<ServiceInstance>` - This predicate is used to filter out `ServiceInstances` which the `DiscoveryClient` might encounter which by forehand you know will not handle any command messages. This might be useful if you've got several services within the Spring Cloud Discovery Service set up which you do not want to take into account for command handling, ever.
-* A "consistent hash change listener" of type `ConsistentHashChangeListener` - Adding a consistent hash change listener provides you the opportunity to perform a specific task if new members have been added to the known command handlers set.
+## Sending Commands between nodes
 
-The `SpringHttpCommandBusConnector` requires three parameters for creation:
+The `CommandBusConnector` is in charge to send the commands, based on a given route, from one node to another.
+This extension to that end provides the `SpringHttpCommandBusConnector`, which uses plain REST for sending commands.
 
-* A "local command bus" of type `CommandBus` - This is the command bus implementation that dispatches commands to the local JVM. These commands may have been dispatched by instances on other JVMs or from the local one.
-* A `RestOperations` object to perform the posting of a command message to another instance.
-* Lastly a "serializer" of type `Serializer` - The serializer is used to serialize the command messages before they are sent over the wire.
+There are three hard requirements when creating this service and one optional configuration:
 
-> **Note**
->
-> The Spring Cloud connector specific components for the `DistributedCommandBus` can be found in the `axon-distributed-commandbus-springcloud` module.
+1. Local `CommandBus` - This "local segment" is the command bus which dispatches commands into the local JVM. It is thus invoked when the `SpringHttpCommandBusConnector` receives a command from the outside, or if it receives a command which is meant for itself.
+2. `RestOperations` - The service used to POST a command message to another instance. In most situations the `RestTemplate` is used for this.
+3. `Serializer` - The serializer is used to de-/serialize the command messages before they are sent over or when they are received.
```

Comment:
```suggestion
3. `Serializer` - The serializer is used to deserialize the command messages before they are sent over or when they are received.
```

Reply:
I'll rephrase this sentence, as I intend to point out both serialization and deserialization are done